frontend/src/pbd-app.html

```diff
@@ -144,9 +144,15 @@
         if (robot === 'pr2') {
           this.set('params.robot', robot);
           this.set('params.hasTwoArms', true);
+          this.set('params.manualFreeze', true);
```

Comment:
Add a comment like "manualFreeze is true for robots that require buttons in Rapid PbD to relax/freeze the arm. Set manualFreeze to false for robots like the Baxter that can activate gravity compensation by just grasping the wrist."